def raise_exception_msg(message=""):
    raise nameError(message)